### ISSUE TITLE
Spurious "Query terminated by the `rethinkdb.jobs` table"

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#4819]
![screenshot from 2015-09-09 15 28 07](https://cloud.githubusercontent.com/assets/505365/9775841/6b4696ae-5707-11e5-9653-1f556a1a3c86.png)

Unless someone else connected to my server (newton with offset 23) and killed it through the `jobs` table, I think the query simply timed out. I definitely didn't use `rethinkdb.jobs` myself.

**Comment by danielmewes** [comment]
@Tryneus could you look into this please?
It seems to happen every time a query times out in the web UI.

**Comment by Tryneus** [comment]
I'll look into this as soon as I'm done preparing the no-protobuf review.  Pretty sure last time I touched this code I got a little over-zealous with suspecting the jobs table.

**Comment by kulbirsaini** [comment]
This is happening for me as well. But not via Admin panel. 

```javascript
Unhandled rejection ReqlRuntimeError: Query terminated by the `rethinkdb.jobs` table in:
r.db("test").table("test_users")
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    at Connection._processResponse (/home/saini/xxx/private/xxxx/xxxx/node_modules/jamadar/node_modules/rethinkdbdash/lib/connection.js:241:21)
    at Socket.<anonymous> (/home/saini/xxx/private/xxxx/xxxx/node_modules/jamadar/node_modules/rethinkdbdash/lib/connection.js:150:14)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:523:20)
```

Rethinkdb 2.1.3

**Comment by danielmewes** [comment]
@kulbirsaini That's interesting. Thanks for letting us know!

**Comment by neumino** [comment]
I also see that in rethinkdbdash/thinky tests.

If I open a feed, and then close it, the response for the `STOP` query is an error:
```
ReqlRuntimeError: Query terminated by the `rethinkdb.jobs` table in:
r.db("test").table("test").changes()
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

It happens with 2.1.3, but not 2.1.2

**Comment by neumino** [comment]
There's a pending `CONTINUE` query too.

**Comment by Tryneus** [comment]
Fix is up in review 3245.

**Comment by Tryneus** [comment]
The fix has been merged to `next` in commit 6f6ecad, and cherry-picked into `v2.1.x` in commit e52f859.  Will be in release 2.1.5.
